internal/artifacts/manifest.go

```diff
@@ -34,6 +35,18 @@ const (
 	requiredEntry  = "MTA-Requires"
 	resourceEntry  = "MTA-Resource"
 	dirContentType = "text/directory"
+
+	moduleMsgProperty   = "module"
+	resourceMsgProperty = "resource"
+
+	wrongArtifactPathMsg      = `failed to generate the manifest file when getting the artifact path of the "%s" module`
+	unknownContentTypeMsg     = `failed to generate the manifest file when getting the "%s" %s content type`
```

Comment:
Use 2 constants for the messages (one for module and one for resource) instead of constructing it from sentence parts

Reply:
done